packager/react-packager/src/DependencyResolver/fastfs.js

```diff
@@ -297,7 +297,7 @@ class File {
     Object.keys(this.children).forEach(key => {
       const file = this.children[key];
       if (file.isDir) {
-        files.push(...file.getFiles());
+        file.getFiles().forEach((f) => files.push(f));
```

Comment:
> Node won't apply more than 65536 arguments to a function

wow good find. I would worry about calling concat many times since it could result in O(n^2) performance. Maybe change `files` to `fileArrays` and build up an array of arrays that get merged at the end with `[].concat(...fileArrays)`